src/Services/ApiProvider/ConfigurationApiProvider.php

```diff
@@ -74,6 +77,7 @@ public function getConfigurationEntry(string $key, ?string $subKey = null): Conf
         $response = $configurationEntryApi->getConfigurationEntryCollection(
             $key,
             null,
+            $value,
```

Comment:
so the new config SDK introduced this extra param? I just hope we're not using it directly in any of the project :)